packages/ratatouille.lv2/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=Ratatouille.lv2
+pkgname=${_name,,}
+pkgver=0.4
+pkgrel=1
+pkgdesc='A guitar amp neural model and impulse response loader and mixer LV2 plugin'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(BSD-3-Clause)
+groups=(lv2-plugins pro-audio)
+depends=(cairo fftw glibc gcc-libs libsndfile libx11)
+makedepends=(lv2 xxd)
+checkdepends=(lv2lint)
+optdepends=('lv2-host: for loading the LV2 plugin')
+source=("https://github.com/brummer10/$_name/releases/download/v$pkgver/$_name-v$pkgver-src.tar.xz")
+sha256sums=('35a072473a9c6d354c1d9a01f710497690c53684a5ff83d6654f41828983bd7a')
+_plugin_uri="urn:brummer:ratatouille"
+
+build() {
+  cd $_name-v$pkgver
+  make
+}
+
+check() {
+  cd $_name-v$pkgver
+  lv2lint -M pack -s '_Z*' -I bin/$_name "$_plugin_uri"
```

Comment:
I had to add `-s'_Z*'` to the lv2lint command because in the in the CI build the plugin shared lib exposes some internal symbols and the [check fails](https://ci.cbix.de/osam-cologne/archlinux-proaudio/1926/1/4) without ignoring those. This isn't the case when I build the packageon my local system. I'm not sure why. I compared the C(XX)/LDFLAGS and they're the same (except I have `-march=native -mtune=native`.

These are the values in the CI.

```
CFLAGS: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         
-Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security -fstack-clash-protection 
-fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer
-g -ffile-prefix-map=/tmp/build/ratatouille.lv2/src=/usr/src/debug/ratatouille.lv2 -flto=auto

CXXFLAGS: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         
-Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security -fstack-clash-protection 
-fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -Wp,-D_GLIBCXX_ASSERTIONS
-g -ffile-prefix-map=/tmp/build/ratatouille.lv2/src=/usr/src/debug/ratatouille.lv2 -flto=auto

LDFLAGS: -Wl,-O1 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now          -Wl,-z,pack-relative-relocs -flto=auto

LTOFLAGS: -flto=auto
```

Reply:
Without `-s '_Z*'` I'm seeing the same issue as on CI with `makechrootpkg` (clean chroot), but not with `makepkg`

Reply:
Very odd.